### DISCUSSION TITLE
Option Names wrong

### DISCUSSION THREAD

**Discussion by Nivatius** [jplag/JPlag#260] [Q&A] [answered]
Hello,
I cant get the options to work.
I looked in the source code and found, that -n is now called -m for example.
This is still displayed differently in the readme file. -h is an unkown option.

**Comment by tsaglam** [comment, marked as answer]
I assume you are using the legacy version of JPlag? Meaning below version 3.0.0?
In that case please refer to the readme of the [legacy branch](https://github.com/jplag/JPlag/tree/legacy).
The current readme also states: 
> Note that the [legacy CLI](https://github.com/jplag/jplag/blob/legacy/README.md) is varying slightly.

@Nivatius please let me know if that solved your issues.

**Comment by Nivatius** [comment]
sorry for the issue, yes the latest release seems to be legacy indeed. So am I right to assume that you don't provide a .jar for the current version?

**Reply by tsaglam** [reply]
Update: We just released v3.0.0 yesterday!

**Comment by tsaglam** [comment]
No problem, glad I could help! We are currently working on releasing [v3.0.0](https://github.com/jplag/JPlag/milestone/1), which will be hopefully soon. There are some domain issues we need to resolve, but then we will release it here and on maven central. You can always build your own pre-release version as described in the readme. However, for a stable version, I would refer to a release.
